Update release notes and CHANGELOG based on 18.9 changes (#12903)
woocommerce/woocommerce-ios: CHANGELOG.md

<!--
Contains editorialized release notes. Raw release notes should go into `RELEASE-NOTES.txt`.
-->
## 18.9

> Stay ahead with our latest WooCommerce update! Explore our new Watch App for instant store data and order updates at a glance. We've fixed the issue with shipping methods, so you can now create and update orders seamlessly. Plus, you can now view, add, edit, or remove multiple shipping lines on orders, enhancing your order management.

Reviewer: ```suggestion
Stay ahead with our latest WooCommerce update! Explore our new Watch App for instant store data and order updates at a glance. We've fixed the issue with shipping methods, plus you can now have multiple shipping lines on orders, so you can handle complex shipping needs for your orders seamlessly.
```
